frontend/public/components/utils/resource-log.tsx

```diff
@@ -241,150 +269,294 @@ export const LogControls: React.FC<LogControlsProps> = ({
-            )}
-          >
-            <span className="text-muted">{label}</span>
-          </Tooltip>
-        )}
+        </React.Fragment>
+      );
+    });
+
+  const label = t('public~Debug container');
+  const renderLogViewerSearch = () => (
+    <>
+      <div className="co-toolbar__item">
+        <LogViewerSearch
+          onFocus={() => {
+            if (status === STREAM_ACTIVE) {
+              toggleStreaming();
+            }
+          }}
+          placeholder="Search"
+          minSearchChars={0}
+        />
       </div>
-      <div
-        className="co-toolbar__group co-toolbar__group--right co-toolbar__group--right"
-        data-test="log-links"
-      >
-        <div className="pf-l-flex">
-          {!_.isEmpty(podLogLinks) &&
-            _.map(_.sortBy(podLogLinks, 'metadata.name'), (link) => {
-              const namespace = resource.metadata.namespace;
-              const namespaceFilter = link.spec.namespaceFilter;
-              if (namespaceFilter) {
-                try {
-                  const namespaceRegExp = new RegExp(namespaceFilter, 'g');
-                  if (namespace.search(namespaceRegExp)) {
-                    return null;
-                  }
-                } catch (e) {
-                  // eslint-disable-next-line no-console
-                  console.warn('invalid log link regex', namespaceFilter, e);
-                  return null;
+      {showDebugAction(resource, containerName) && !isWindowsPod(resource) && (
+        <Link
+          to={`${resourcePath(
+            'Pod',
+            resource.metadata.name,
+            resource.metadata.namespace,
+          )}/containers/${containerName}/debug`}
+        >
+          {label}
+        </Link>
+      )}
+      {showDebugAction(resource, containerName) && isWindowsPod(resource) && (
+        <Tooltip
+          content={t('public~Debug in terminal is not currently available for windows containers.')}
+        >
+          <span className="text-muted">{label}</span>
+        </Tooltip>
+      )}
+    </>
+  );
+
+  return (
+    <Toolbar>
+      {isMobile ? (
+        <ToolbarContent>
+          <ToolbarGroup alignment={{ default: 'alignLeft' }}>
+            <ToolbarItem>
+              {showStatus()}
+              <Split hasGutter>
+                <SplitItem isFilled>{dropdown && dropdown}</SplitItem>
+                <SplitItem isFilled>{logTypeSelect(!hasPreviousLog)}</SplitItem>
+              </Split>
+              {renderLogViewerSearch()}
+            </ToolbarItem>
+          </ToolbarGroup>
+          <ToolbarGroup alignment={{ default: 'alignRight' }}>
+            <ToolbarItem>
+              <Dropdown
+                className="pf-c-dropdown"
+                position="right"
+                isOpen={isMenuToggleOpen}
+                onSelect={onSelect}
+                toggle={
+                  <MenuToggle
+                    ref={toggleRef}
+                    aria-label="kebab dropdown toggle"
```

Comment:
Need to internationalize this string and make it something meaningful.  Maybe the following?
```suggestion
                    aria-label={t('public~Options')}
```